src/widgets/dropdown/widget-dropdown-view.js

```diff
@@ -93,19 +88,19 @@ module.exports = cdb.core.View.extend({
   _togglePinned: function () {
     var pinned = !this.model.get('pinned');
```

Comment:
Why not render the component when any of those attributes have changed within the model?